code/modules/clothing/glasses/misc.dm

```diff
@@ -66,7 +66,7 @@
 	body_parts_covered = 0
 
 /obj/item/clothing/glasses/sunglasses
-	desc = "Strangely ancient technology used to help provide rudimentary eye cover. Enhanced shielding reduces many flashes."
+	desc = "Strangely ancient technology used to help provide rudimentary eye cover. Doesn't really protects from flashes, but stylish."
```

Comment:
This is wrong. Regular sunglasses do provide flash protection. What they don't provide is flash immunity. Pairing sunglasses with a helmet makes you flash immune.

SecHUDs also only provide limited protection, the same level regular sunglasses do. Consider accounting for this.

Reply:
Yeah, there might be a better way to express that but the "Doesn't really protect" sort of mean "flash protection but not flash immunity".